bin/get_ion_annotations.py

```diff
@@ -7,7 +7,6 @@
 import pandas as pd
 import numpy as np
 import argparse
-from pyopenms.Plotting import *
```

Comment:
Was this intentional?

Reply:
Yes, that's not a part of pyopenms 3.0.0 anymore. It was also not used in the code, it was an unused import